From 91ff757ce73976f7434b6e03e33a6f933411cd0d Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Fri, 18 Jan 2019 09:23:59 -0700
Subject: [PATCH 12/12] drm: rcar-du: Fix build error

On arm{32,64} allyesconfig builds:

drivers/gpu/drm/rcar-du/rcar_dw_hdmi.c:40:1: error: incomplete result type 'enum drm_mode_status' in function definition
drivers/gpu/drm/rcar-du/rcar_dw_hdmi.c:47:10: error: incomplete definition of type 'struct drm_display_mode'
drivers/gpu/drm/rcar-du/rcar_dw_hdmi.c:48:10: error: use of undeclared identifier 'MODE_CLOCK_HIGH'
drivers/gpu/drm/rcar-du/rcar_dw_hdmi.c:50:9: error: use of undeclared identifier 'MODE_OK'

After the removal of drmP.h from this file and its removal from
bridge/dw_hdmi.h, these types are not defined anymore, as they
were implicitly coming into that file through drm_crtc.h.

Fixes: 428747ae5cca ("drm: remove include of drmP.h from bridge/dw_hdmi.h")
Fixes: 9a47db8e7a9d ("drm: rcar-du: Remove inclusion of drmP.h")
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 drivers/gpu/drm/rcar-du/rcar_dw_hdmi.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/rcar-du/rcar_dw_hdmi.c b/drivers/gpu/drm/rcar-du/rcar_dw_hdmi.c
index 0fb25113f19c..452461dc96f2 100644
--- a/drivers/gpu/drm/rcar-du/rcar_dw_hdmi.c
+++ b/drivers/gpu/drm/rcar-du/rcar_dw_hdmi.c
@@ -12,6 +12,7 @@
 #include <linux/platform_device.h>
 
 #include <drm/bridge/dw_hdmi.h>
+#include <drm/drm_modes.h>
 
 #define RCAR_HDMI_PHY_OPMODE_PLLCFG	0x06	/* Mode of operation and PLL dividers */
 #define RCAR_HDMI_PHY_PLLCURRGMPCTRL	0x10	/* PLL current and Gmp (conductance) */
-- 
2.20.1

